packages/dapp/src/components/Widgets/Widgets.tsx

```diff
@@ -81,38 +89,40 @@ export function Widgets() {
       showWelcome={!welcomeScreenEntered}
       handleGetStarted={handleGetStarted}
     >
-      {import.meta.env.MODE === 'testnet' && (
-        <Grid item xs={12} mt={theme.spacing(6)}>
-          <TestnetAlert />
-        </Grid>
-      )}
-      <WidgetContainer
-        onClick={handleGetStarted}
-        showWelcome={!welcomeScreenEntered}
-        isActive={_starterVariant === TabsMap.Exchange.value}
-      >
-        <Widget starterVariant={TabsMap.Exchange.value as WidgetSubvariant} />
-      </WidgetContainer>
-      <WidgetContainer
-        onClick={handleGetStarted}
-        showWelcome={!welcomeScreenEntered}
-        isActive={_starterVariant === TabsMap.Refuel.value}
-      >
-        <Widget starterVariant={TabsMap.Refuel.value as WidgetSubvariant} />
-      </WidgetContainer>
-      {import.meta.env.VITE_ONRAMPER_ENABLED ? (
+      <>
```

Comment:
Do we need this here? 🙂

Reply:
Maybe not? Maybe, it´s the same as with those TemplateStringArrays which can be resolved by adjustments in the typescript config? For me, it resolved the following error for the moment:

![image](https://github.com/lifinance/jumper.exchange/assets/43956540/5ff2d84b-ee32-4a46-91ee-497d365d8e41)

Reply:
It was resolved with `React.FC<PropsWithChildren<{}>>`